x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/telemetry/alerting_and_actions_telemetry.ts

```diff
@@ -245,49 +213,44 @@ export default function createAlertingAndActionsTelemetryTests({ getService }: F
       );
 
       expect(
-        telemetry.count_connector_types_by_action_run_outcome_per_day['test.throw'].failure
-      ).to.greaterThan(0);
+        telemetry.count_connector_types_by_action_run_outcome_per_day['test.throw'].failure > 0
+      ).to.be(true);
     }
 
     function verifyAlertingTelemetry(telemetry: any) {
       logger.info(`alerting telemetry - ${JSON.stringify(telemetry)}`);
       // total number of enabled rules
-      expect(telemetry.count_active_total).to.equal(18);
+      expect(telemetry.count_active_total).to.equal(9);
 
       // total number of disabled rules
       expect(telemetry.count_disabled_total).to.equal(3);
 
       // total number of rules broken down by rule type
-      expect(telemetry.count_by_type.test__noop).to.equal(6);
-      expect(telemetry.count_by_type['example__always-firing']).to.equal(3);
-      expect(telemetry.count_by_type['test__cumulative-firing']).to.equal(3);
+      expect(telemetry.count_by_type.test__noop).to.equal(3);
+      expect(telemetry.count_by_type.test__patternFiring).to.equal(3);
       expect(telemetry.count_by_type.test__multipleSearches).to.equal(3);
-      expect(telemetry.count_by_type.test__onlyContextVariables).to.equal(3);
       expect(telemetry.count_by_type.test__throw).to.equal(3);
 
       // total number of enabled rules broken down by rule type
-      expect(telemetry.count_active_by_type['example__always-firing']).to.equal(3);
-      expect(telemetry.count_active_by_type['test__cumulative-firing']).to.equal(3);
+      expect(telemetry.count_active_by_type.test__patternFiring).to.equal(3);
       expect(telemetry.count_active_by_type.test__multipleSearches).to.equal(3);
-      expect(telemetry.count_active_by_type.test__noop).to.equal(3);
-      expect(telemetry.count_active_by_type.test__onlyContextVariables).to.equal(3);
       expect(telemetry.count_active_by_type.test__throw).to.equal(3);
 
       // throttle time stats
       expect(telemetry.throttle_time.min).to.equal('0s');
-      expect(telemetry.throttle_time.avg).to.equal('115.5s');
-      expect(telemetry.throttle_time.max).to.equal('600s');
+      expect(telemetry.throttle_time.avg).to.equal('0s');
```

Comment:
Seems like we should provide some throttle values in the new set of rules, so we can see some non-zero numbers.

Reply:
Sure, I can add throttle values to one of the rules!

Reply:
Resolved in this commit [a273697](https://github.com/elastic/kibana/pull/161096/commits/a273697bcdba1a561aea33d302ffdccdb4ff8dfe)